src/cloud/project/project-webint-snap.md

```diff
@@ -140,7 +144,7 @@ To create a database dump:
  {:.bs-callout-info}
 
 -  We recommend putting the application in maintenance mode before doing a database dump in Production environments.
--  The command creates an archive in your local project directory called  `dump-<timestamp>.sql.gz`.
+-  The command creates an archive in your local project directory called  `dump-<label><timestamp>.sql.gz`.
```

Comment:
Is <label> the name of the database that is dumped?

Reply:
Yes
'main', 'sales' or 'quote'